src/css/application.css

```diff
@@ -73,6 +73,75 @@ body {
   font-family: 'Roboto', sans-serif;
 }
 
+/** @define identity-confirmation */
+
+.identity-confirmation {
+  background: rgba(0, 0, 0, 0.48);
+  height: 100vh;
+  position: absolute;
+  width: 100vw;
+  z-index: 3;
+}
+
+.identity-confirmation__confirm-button {
+  background-color: #2ecc71;
+  border-radius: 2px;
+  border-width: 0;
+  box-shadow: 0 1px 4px rgba(0, 0, 0, 0.6);
+  color: #ecf0f1;
+  cursor: pointer;
+  display: block;
+  margin: 30px auto;
+  outline: none;
+  overflow: hidden;
+  padding: 0;
+  padding: 1em;
+  position: relative;
+  transition: background-color 0.3s;
+}
+
+.identity-confirmation__confirm-button:hover,
+.identity-confirmation__confirm-button:focus {
+  background-color: #27ae60;
+}
+
+.identity-confirmation__confirm-button::before {
+  background-color: rgba(236, 240, 241, 0.3);
+  border-radius: 100%;
+  content: "";
+  display: block;
+  left: 50%;
+  padding: 0;
+  position: absolute;
+  top: 50%;
+  transform: translate(-50%, -50%);
+  width: 0;
+}
+
+.identity-confirmation__confirm-button:active::before {
+  padding-top: 120%;
+  transition: width 0.2s ease-out, padding-top 0.2s ease-out;
+  width: 120%;
+}
```

Comment:
Regarding all the button CSS rules, it's to imitate the Google Material Design feel. Thought it fit in with the look and feel of popcode. Lmk if you think it should go in a different direction.

[Check it out here!](https://cl.ly/nEAF)